pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -346,6 +346,10 @@ def on_train_end(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule") -
         self._save_last_checkpoint(trainer, monitor_candidates)
         trainer.fit_loop.global_step += 1
 
+        # notify loggers
+        if trainer.is_global_zero and trainer.logger:
+            trainer.logger.after_save_checkpoint(proxy(self))
+
```

Comment:
could you add a test for this to confirm this bug doesn't happen again?

Reply:
I added the test.